views/js/qtiCreator/editor/widgetToolbar.js

```diff
@@ -383,11 +384,11 @@ define([
                 buttonConfig = handleConfig(buttonConfig, barConfig.extendDefaultConfig);
                 barConfig = $.extend(defaultBarConfig, (barConfig || {}));
 
-
                 toolbar = buildToolbar(buttonConfig, target, tlbWrapper);
 
-                if(barConfig.title){
-                    title = $('<span>', {text : barConfig.title, 'class' : 'tlb-title', title : barConfig.title});
+                if (barConfig.title) {
+                    const getQtiClassTitle = qtiClass => classTitles[qtiClass] || qtiClass;
```

Comment:
I didn't want to repeat myself, but apparently I need to.

Is it an `es6` file?
I believe we shouldn't mix `es5` and `6` syntax in a single file.